src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<p>I would maybe remove this in favor of explaining what the overall goal is (What is the business use case?)</p><p> </p><p>The tutorial leverages the following Google Cloud ML services: </p><p> </p><p>-&gt; This tutorial uses the following Google Cloud products:</p><p>      (add BigLake)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>